### PR TITLE
Disable Beta Popup Dialog

### PR DESCRIPTION
**Pull request by ani-kalpachka**
Disabled Beta Popup Dialog:

image

The component is not deleted in case we need it back. So far its import is commented.
Removed the unused imports in the component.

### CODE REVIEW

**Comment by imilchev** [comment]
The Playwright tests are actually testing the pop up as well. Now that it is gone, they fail. Try to fix them. It should be as easy as removing the lines that actually interact with the popup. 👀

File: e2e/staging/homepage.spec.ts [hunk 1]
```diff
@@ -10,19 +10,9 @@ test('test homepage on staging', async ({ page }) => {
...
 
-  // Click text=Това е бета версията на платформата на Подкрепи.бг преди предстоящия наесен офиц
-  await page
-    .locator(
-      'text=Това е бета версията на платформата на Подкрепи.бг преди предстоящия наесен офиц',
-    )
-    .click()
-
   // Check input[type="checkbox"]
   await page.locator('input[type="checkbox"]').check()
```
Comment (kachar) [inline e2e/staging/homepage.spec.ts:15]: Let's remove this checkbox click as it's part of the beta popup

File: src/components/index/IndexPage.tsx [hunk 2]
```diff
@@ -20,7 +18,7 @@ export default function IndexPage() {
       disableGutters
       title={t('index:title2')}
       metaDescription={t('index:metaDescription')}>
-      <BetaPopUpDialog />
+      {/* <BetaPopUpDialog /> */}
```
Comment (kachar) [inline src/components/index/IndexPage.tsx:21]: Let's remove the component, we can always bring it back from git history

File: src/components/index/sections/BetaPopUpDialog.tsx [hunk 3]
```diff
@@ -11,12 +11,8 @@ import {
   FormControlLabel,
   Typography,
 } from '@mui/material'
```
Comment (kachar) [inline src/components/index/sections/BetaPopUpDialog.tsx:13]: Let's remove the component, we can always bring it back from git history

**Review by imilchev — APPROVED** [review]

**Pull request merged**